radio/src/targets/common/arm/stm32/flysky_gimbal_driver.cpp

```diff
@@ -263,24 +265,27 @@ bool flysky_gimbal_init()
 {
   if (flysky_gimbal_init_uart() != 0) return false;
 
-  // Wait 70ms for FlySky gimbals to respond. According to LA trace, minimally
-  // 23ms is required
-  _fs_gimbal_detected = false;
```

Comment:
This doesn't seem to be initialised anywhere now. This change should probably be removed.